CHANGELOG.md

```diff
@@ -338,13 +344,11 @@ designing and testing several of these features.
 
 ## Older Changelogs, from individual modules:
 
-- [Accoutrement Init][io]
 - [Accoutrement Color][co]
 - [Accoutrement Layout][lo]
 - [Accoutrement Scale][so]
 - [Accoutrement Type][to]
 
-[io]: https://github.com/oddbird/accoutrement-init/blob/master/CHANGELOG.md
```

Comment:
Should we keep these, since they're just linking to historic documentation? I don't have a strong feeling either way.